var/spack/repos/builtin/packages/jsonnet/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Jsonnet(MakefilePackage):
+    """A data templating language for app and tool developers based on JSON"""
+
+    homepage = "https://jsonnet.org/"
+    git = "https://github.com/google/jsonnet.git"
+
+    maintainers = ["jcpunk"]
+
+    version("develop", branch="master")
```

Comment:
Can we call this version `master` to match the branch name?

Reply:
I can fix that, it was in the thing I copied from.